CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/svg/SVGGraphBuilder.java

```diff
@@ -217,23 +217,28 @@ public SVGGraphBuilder fromPerspective(final AxisConstants exportPerspective) {
     public SVGData build() throws InterruptedException {
         
         final SVGObject svgGraph = SVGTemplateConstants.LAYOUT.getSVGObject();
-        
-        try{
-            preBuild();
-        } catch (final IllegalArgumentException ex){
-            LOGGER.log(Level.SEVERE, ex.getLocalizedMessage(), ex);
-            return null;
+        try {
+            try {
+                preBuild();
+            } catch (final IllegalArgumentException ex){
+                LOGGER.log(Level.SEVERE, ex.getLocalizedMessage(), ex);
+                return null;
+            }
+
+            // Build the SVG image
+            buildHeader(svgGraph);
+            buildNodes(svgGraph);
+            buildConnections(svgGraph);            
+            buildLayout(svgGraph);
+
+            // Clean up the builder
+            postBuild();     
+        } catch (final InterruptedException ex) {
+            // This class utilizes GraphVisualAccess and is responsible for releasing the graph lock when the plugin is interupted
+            postBuild();
+            throw ex;
         }
```

Comment:
Looks like you want the postBuild() to be run at the end in all cases.
Could this be done as a `finally` block at the end a single try statement, without needing to nest an extra try block ?

Reply:
I'll have a play with it and get back to you. As long as the postBuild() in a finally block doesn't cause issues with the catch rethrowing the exception we should be good